code/modules/mob/living/simple_animal/slime/powers.dm

```diff
@@ -99,6 +99,14 @@
 			return FALSE
 		to_chat(src, "<span class='warning'><i>Another slime is already feeding on this subject...</i></span>")
 		return FALSE
+
+	if(iscarbon(M))
+		var/mob/living/carbon/C = M
+		if((C.wear_suit && ((C.wear_suit.flags & THICKMATERIAL)) && ((C.head && ((C.head.flags & THICKMATERIAL))))))
+			if(silent)
```

Comment:
I'd just make this into a `!silent` and put the to_chat in the if scope

Reply:
I'd rather keep it the same as the other conditions.